### PR TITLE
[Xcode] Build swift bindings as part of llbuild.framework

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
Depends on https://github.com/apple/swift-llbuild/pull/247

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
I can document that in docs/development.rst

File: Xcode/Configs/Common.xcconfig [hunk 1]
```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "Version.xcconfig"
```
Comment (dmbryson) [inline Xcode/Configs/Common.xcconfig]: Perhaps rebase on master?  This line shouldn't be necessary and it would be nice to be sure it doesn't merge oddly.

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by jakepetroules** [comment]
@swift-ci please smoke test

**Comment by dmbryson** [comment]
Where do we stand here?  Should we merge this now?

**Comment by ddunbar** [comment]
I think @aciidb0mb3r is tracking what more needs to be done

**Pull request merged**